src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.scss

```diff
@@ -20,24 +24,28 @@
   font-weight: 300;
 }
 
+.previouslyViewedVersionedStableId {
```

Comment:
There are things in the name of this class that aren't relevant from the perspective of CSS. The word "versioned", for instance certainly can be omitted. But I am wondering whether the `previouslyViewed` part is significant at all — won't other bookmarks, if we ever have them, be formatted in a similar fashion?

Perhaps `objectId` and `objectType` (or `entityId` and `entityType`) would work better here than `previouslyViewedVersionedStableId` and `previouslyViewedType`?